metadata.rb

```diff
@@ -3,9 +3,9 @@
 maintainer_email  'james@conant.io'
 license           'MIT'
 description       'Installs Sonarr.'
-version           '1.0.4'
+version           '1.1.0'
```

Comment:
I realize this is a bit nit-picky, but in changing node attributes to be based on strings rather than symbols we are breaking compatibility with older configs. In light of that, we should update the version to `2.0.0`.

Reply:
Additionally, if features in here require v13 of Chef we should update the major version.

Reply:
Ah, fair point!